docs/tutorials/mnist.ipynb

```diff
@@ -873,32 +874,44 @@
     },
     {
       "cell_type": "code",
+      "execution_count": 27,
       "metadata": {
+        "colab": {},
         "colab_type": "code",
-        "id": "n8vuQpSLlBV2",
-        "colab": {}
+        "id": "n8vuQpSLlBV2"
       },
+      "outputs": [],
       "source": [
         "EPOCHS = 3\n",
         "BATCH_SIZE = 32\n",
+        "\n",
         "NUM_EXAMPLES = len(x_train_tfcirc)"
-      ],
-      "execution_count": 0,
-      "outputs": []
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# For testing\n",
```

Comment:
This will display in the site documentation/Colab ?
Can you search/replace for the test instead of surfacing this infra to users?

Reply:
Done in "scripts/test_tutorials" replace `NUM_EXAMPLES=.*` with `NUM_EXAMPLES=10`.

Reply:
Much better. Thanks!